changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+v?.?.? (2024-04-04)
```

Comment:
We need to decide on the versioning as the tagged version on master is `0.1`.

Reply:
Good point, I should rénale the tag to 0.0.1.